### ISSUE TITLE
The next version

### ISSUE THREAD

**Issue by toyeebgodo** [Qolzam/react-social-network#42]
@Qolzam, you mentioned here (https://github.com/Qolzam/react-social-network/issues/15) that "**The next version of this project migrate to typescript with many new awesome stuff, cleaner code, high performance, with AWS and Firebase back-end, easy deploying and CLI to modify project easily,"** and you also said here (https://github.com/Qolzam/react-social-network/issues/36) that , **". . . In the new version I have added both reset password and User confirmation #23. I'm trying to make time to release this update."** 

Given that releasing this projects as open source is big-time charity on your part and your other charitable project contributions (such as this: https://github.com/Qolzam/react-mobile-social ) to the developer community has been nothing less than magnanimous, my humble question is, how's the work on the new release coming along? Any estimation as to when you will be releasing it? Any other information you care to share? Thank you as always for your immense contribution to my life as a developer.

**Comment by QuantumInformation** [comment]
It will be ready when it is ready.

**Comment by Qolzam** [comment]
I appreciate your consideration. Regardless of being too busy nowadays, There are some issues need to be considered more as the next version is completely different with demo code. To be honest for the next version I've spent more time to have a reliable code (I've tried my best!).

It would be more logical than current version in data structure and performance. I have changed the code many times to support variety of developers. 

The good point is, the next version is terribly flexible in adding new feature and developing on your own opinion. 

Almost 90 percent of the project is supporting type definition of TypeScript, no dependency on firebase, use [DI](https://en.wikipedia.org/wiki/Dependency_injection) concept to make independent modules, Hot reloading, show nice compile error in the browser, reset password, forget password, OAuth, auto scroll loading for posts and people page, better performance and so on. 

Sure there are more things to do but time is short and right now I'm the only person who is developing this project. 

Hope it works as I expect and I find some contributors to share this work with them. As a result I could work some stuff such as [mobile app](https://github.com/Qolzam/react-mobile-social) or GraphQL.
